docs/topics/proc_getting-launcher-url-and-credentials-on-openshiftlocal.adoc

```diff
@@ -6,13 +6,13 @@
 
 
 [#getting-launcher-url-and-credentials-on-openshiftlocal_{context}]
-= Getting {launcher} URL and Credentials on {OpenShiftLocal}
+= Getting the {launcher} Service URL and Credentials on {OpenShiftLocal}
```

Comment:
This warrants changing the anchor ID to match, in my opinion, but we can address this later.

Same with every other heading changed.